backend/src/test/scala/cromwell/backend/standard/StandardValidatedRuntimeAttributesBuilderSpec.scala

```diff
@@ -75,7 +75,18 @@ class SharedFileSystemValidatedRuntimeAttributesBuilderSpec extends WordSpecLike
       message should include("Unrecognized runtime attribute keys: docker")
     }
 
-    "log a warning and fail to validate an invalid Docker entry" in {
+    "log a warning and validate an invalid Docker entry" in {
+      /*
+      NOTE: The behavior used to be: when present, a "docker" runtime attribute would always be validated to ensure that
+      the value of the runtime attribute was a String-- even if the actual runtime attribute was unsupported by the
+      backend! NOW, if the backend doesn't support docker, and the runtime attributes contain say as an invalid integer,
+      say `{ docker: 1 }`, the validation will now succeed. If we want to change this, the git history contains a
+      concept of validated-yet-still-unsupported attributes. One could use this behavior to validate other standard
+      attributes such as CPU, Memory, etc.-- checking their syntax even when they are unsupported by the current
+      backend.
```

Comment:
I think that's fine since Docker is ultimately a backend-specific attribute. AFAIK only memory has a specific format specified in the WDL spec (no idea why...)

So if ChrisBackend wants `docker` to custom validation, it should be able to.